src/components/ListPaginationTop.tsx

```diff
@@ -0,0 +1,27 @@
+import { Box, Typography } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+
+interface ListPaginationTopProps {
+  count: number;
+  rowsPerPage: number;
+  page: number;
+}
+
+export const ListPaginationTop = ({ count, rowsPerPage, page }: ListPaginationTopProps) => {
```

Comment:
Dette er nå duplisering av det som ligger i `ListPagination`. Er det mulig å same det i en komponent?

Om denne feks endres til noe sånn som `<ListPaginationSummary start={...} end={...} total={...}>`

Så kan den brukes for både Cristin- og NVA-søk, altså det vil virke uavhengig av om paginering bruker `page`, `offset`, eller `from` som jeg tror er de tre(!) ulike måtene vi håndterer paginering på i dag 🤔 

Da kan den brukes direkte i `ListPagination`, og vi kan legge den til i toppen alle plasser vi vil, og vi vil være sikre på at de viser det samme begge plasser

Reply:
mulig jeg overkompliserer, men det å ha samme visning implementert to ulike plasser har en tendens til å gi feil en eller annen gang :p

Reply:
En liten moderering: `ListPaginationTop` gir mening, siden det blir en wrapper som høyrestiller, og at det skal inne flere ting her (som sortering).

Kanskje bare en egen komponent `ListPaginationSummary` er tingen, og den kan brukes både her og i `ListPagination`

Reply:
Ikke dumt! Will do